javascripts/govuk/multivariate-test.js

```diff
@@ -71,18 +71,16 @@
   };
 
   MultivariateTest.prototype.setCustomVar = function(cohort) {
-    window._gaq = window._gaq || [];
-    window._gaq.push([
-      '_setCustomVar',
-      this.customVarIndex,
-      this.cookieName(),
-      cohort,
+    GOVUK.analytics.setDimension(
+      this.customVarIndex, 
+      this.cookieName(), 
+      cohort, 
       2 // session level
-    ]);
+    ); 
     // Fire off a dummy event to set the custom var on the page.
     // Ideally we'd be able to call setCustomVar before trackPageview,
     // but would need reordering the existing GA code.
```

Comment:
Does this comment still apply? The new analytics work means custom variables _can_ be set before the initial pageview is tracked.